src/components/shared/AddRemoveForm.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react'
 import Buttons from '../buttons/index'
-import Settings from '../settings/index'
-import utils from '../../utils'
+// import Settings from '../settings/index'
+import utils from 'utils'
 
 class AddRemoveForm extends React.Component {
 
   state = {
     errorMessage: null,
     items: [],
     filteredItems: [],
-    recentOnly: Settings.UserSettings.getSettings().addRemoveRecentOnly || false
+    recentOnly: /*Settings.UserSettings.getSettings().addRemoveRecentOnly ||*/ false
   }
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
I was messing with this while writing tests. I think it could go either way, but we're being eager beavers.